.github/workflows/run-tests.yml

```diff
@@ -12,14 +12,14 @@ defaults:
 
 jobs:
   tests:
-    name: Integration and Unit tests
+    name: Unit tests
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-      - name: Run tests
+      - name: Run unit tests
         run: |
-          cargo test --lib transport
+          cargo test --lib
```

Comment:
@Xavrax   this line works as per our requirement. Isn't it!?

Reply:
Does it omits the `tests/` directory ;o?